### PR TITLE
Enable fullscreen

### PR DESCRIPTION
**Pull request by Torakko**

### CODE REVIEW

File: PixelHunter1995/Main.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ public class Main : Game
         public Main()
         {
             graphics = new GraphicsDeviceManager(this);
+            graphics.IsFullScreen = true;
```
Comment (ravertass) [inline PixelHunter1995/Main.cs:19]: Looking good man

**Review by ravertass — APPROVED** [review]

File: PixelHunter1995/Main.cs (hunk 1 above)
Comment (Hattes) [inline PixelHunter1995/Main.cs:19]: Look **at** _this_ fancy comment!

Comment (Torakko) [reply PixelHunter1995/Main.cs:19]: Thanks :)

**Review by Hattes — APPROVED** [review]

File: PixelHunter1995/Main.cs (hunk 1 above)
Comment (majstroll) [inline PixelHunter1995/Main.cs:19]: How full is a full screen?

**Pull request merged**